tests/pebble.rs

```diff
@@ -230,6 +233,69 @@ async fn replacement_order() -> Result<(), Box<dyn StdError>> {
     Ok(())
 }
 
+/// Test that it is possible to deactivate an order's authorizations
+#[tokio::test]
+#[ignore]
+async fn order_deactivate() -> Result<(), Box<dyn StdError>> {
+    try_tracing_init();
+
+    let env = Environment::new(EnvironmentConfig::default()).await?;
+
+    let idents = dns_identifiers(["authz-deactivate.example.com"]);
+    let new_order = &NewOrder::new(&idents);
+    let mut order = env.account.new_order(new_order).await?;
+
+    // Each of the order's authorizations should start pending.
+    for authz in order.authorizations().await? {
+        assert_eq!(authz.status, AuthorizationStatus::Pending);
+    }
+
+    // With all authz's deactivated, the order should be status == Invalid
+    // However, until https://github.com/letsencrypt/pebble/pull/301 is merged Pebble returns
+    // an unspecified "deactivated" state we reject as a JSON decoding error about an unexpected
+    // status.
+    // TODO(@cpu): when Pebble is fixed, replace with an assertion on order status.
+    let err = order.deactivate().await.unwrap_err();
+    assert!(matches!(err, Error::Json(_)));
+    assert!(err.to_string().contains("deactivated"));
```

Comment:
This is specific to Pebble, Boulder has already been fixed. I'll ping the maintainers during business hours to see about getting the fix merged & a release cut.

Reply:
Ohh, you have a long history with this stuff. 😄

Reply:
The fix was merged upstream after a bump so this workaround fell away.

There's no release yet so I added a commit up-front to build Pebble from src in CI. The time difference is minimal and we can revert once there's a new release w/ updated binaries.